src/Infrastructure/Processing/RedisAssetStatusCache.php

```
@@ -0,0 +1,141 @@
+
+        if ($written === false) {
+            throw RedisAssetStatusCacheException::storeFailed();
+        }
+    }
+
+    private static function connect(string $host, int $port, ?string $password): object
+    {
+        $redisClass = 'Redis';
+
+        if (! class_exists($redisClass)) {
+            throw RedisAssetStatusCacheException::extensionNotAvailable();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Infrastructure\Processing\Exception\RedisAssetStatusCacheException' in method 'connect'. (undefined)

(StaticAccess)